server/sonar-server/src/main/java/org/sonar/server/app/ServerProcessLogging.java

```diff
@@ -77,4 +84,23 @@ private void configureRootLogger(Props props) {
     ServerLogging.configureHardcodedLevels(helper);
   }
 
+  /**
+   * Setup one or more specified loggers to be non additive and to print to System.out which will be caught by the Main
+   * Process and written to sonar.log.
+   */
+  private void configureDirectToConsoleLoggers(LoggerContext context, String... loggerNames) {
+    LogbackHelper.RootLoggerConfig config = newRootLoggerConfigBuilder()
+      .setProcessId(ProcessId.APP)
+      .setThreadIdFieldPattern("")
+      .build();
+    String logPattern = helper.buildLogPattern(config);
+    ConsoleAppender<ILoggingEvent> consoleAppender = helper.newConsoleAppender(context, "CONSOLE", logPattern);
+
+    for (String loggerName : loggerNames) {
+      Logger consoleLogger = context.getLogger(loggerName);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make the "consoleLogger" logger private static final and rename it to comply with the format "LOG(?:GER)?". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1312)